src/pages/cli/project/troubleshooting.mdx

```diff
@@ -0,0 +1,263 @@
+
+1. It generates CloudFormation for deployment of resources to AWS.
+1. If the application contains a GraphQL API, the CLI runs `amplify api gql-compile` internally to compile the schema and generate VTL (Velocity Templates) for mapping resolvers and CloudFormation templates to allocate AWS resources. 
+1. It builds, packages, and uploads the deployment artifacts into the application's deployment bucket. 
+1. It uses the AWS CloudFormation SDK to deploy the CloudFormation stack into your AWS account.
+1. On a successful deployment of the resources to AWS, the Amplify CLI renames the deployment package in the deployment bucket to `#current-cloud-backend.zip`. This file is the source of truth for your deployment.
+1. If your local project is deleted or corrupted, you can always go back to your last successful deployment using [`amplify pull`](/cli/start/workflows/#amplify-pull).
+
+** This table shows the important files and folders in the Amplify deployment bucket **
+
+|Name|Type|Purpose|
+|-|-|-|
+|#current-cloud-backend.zip|zip file|Source of truth for the Amplify project.|
+|amplify-appsync-files/ |folder| GraphQL API deployment artifacts (VTL resolvers, CloudFormation) |
+|amplify-cfn-templates/ |folder| CloudFormation templates generated by Amplify for each resource category |
+|amplify-meta.json| file | Used by CLI core and the plugins to log internal information and to communicate with each other |
+|backend-config.json | file | Configuration about how your project's backend connects to AWS resources | 
+|hooks/ | folder | User provided Amplify hooks scripts |
+|root-cloudformation-stack.json | file | The parent CloudFormation template for your application's nested stack |
+
+
+** The following sections contain examples of different types of "amplify push" failures and suggested resolutions **
+
+### Push failures from unmanaged changes to cloud resources - Drift 
+<Callout warning>
+
+**Warning:** Once an application is deployed to the cloud using Amplify CLI the cloud resources like DynamoDB tables, Cognito Roles, IAM policies must never be updated outside of Amplify CLI or Amplify Studio.
+
+</Callout>
+
+When a resource is deployed through Amplify is updated outside of Amplify this is called an unmanaged change. 
+The Amplify CLI cannot track such changes since they are applied outside of the Cloudformation generated by Amplify to provision those resources. 
+This difference between the expected configuration of the resource in the Cloudformation vs the actual configuration of the service is called a “CloudFormation drift”. 
+Currently there is no automated way to reverse a CloudFormation drift. Each resource must be inspected through the AWS console for drift in it's CloudFormation stack and the drift has to be resolved manually.
+
+Assume, using Amplify CLI you have created a GraphQL schema with multiple models, indexes and resolvers and deployed it to the cloud. 
```

Comment:
```suggestion
Assume, using Amplify CLI, you have created a GraphQL schema with multiple models, indexes and resolvers and deployed it to the cloud. 
```

Reply:
Could you help me understand the purpose of this "Assume...." at all? Aren't you repeating the scenario in "Scenario 1: ..."?